src/view/plancandidate/DialogUploadImage.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+    Box,
+    Button,
+    Center,
+    HStack,
+    Image,
+    Spinner,
+    Text,
+    VStack,
+} from "@chakra-ui/react";
+import { FullscreenDialog } from "src/view/common/FullscreenDialog";
+
+type Props = {
+    visible: boolean;
+    imageURLs: string[];
+    isUploading: boolean;
+    onUploadClick: () => void;
+    onClose: () => void;
+};
+
+const DialogUploadImage = ({
+    visible,
+    isUploading,
+    imageURLs,
+    onUploadClick,
+    onClose,
+}: Props) => {
+    const renderContent = () => {
+        if (isUploading) {
+            return (
+                <Spinner
+                    thickness="4px"
+                    speed="0.65s"
+                    emptyColor="gray.200"
+                    color="#84A6FF"
+                    size="xl"
+                />
+            );
+        } else {
+            return (
+                <VStack
+                    flex={1}
+                    w="100%"
+                    px="16px"
+                    py="16px"
+                    alignItems="center"
+                    spacing="32px"
+                >
+                    <HStack gap="20px" flexWrap="wrap" justifyContent="center">
+                        {imageURLs.map((url, index) => (
+                            <Box
+                                key={index}
+                                w="200px"
+                                h="200px"
+                                borderRadius="20px"
+                                overflow="hidden"
+                            >
+                                <Image
+                                    src={url}
+                                    alt={`選択された画像${index}`}
+                                    width={100}
+                                    height={100}
```

Comment:
おしいです！
100とすると"100px"となってしまうので、以下のようにしてください
```suggestion
                                    width="100%"
                                    height="100%"
```
![image](https://github.com/poroto-app/poroto/assets/55840281/1962bcfa-a99a-44f8-a1fc-cdc96010cc35)